README.md

```diff
@@ -46,11 +46,36 @@ Other devices may be compatible and all that is needed is to add the ID to apex-
 
 ## Installation
 
+For installing this software, follow the steps:
+
 ### UDev
 
-Enter the following data from [here](https://gist.github.com/ToadKing/d26f8f046a3b707e9e4b9821be5c9efc) (Shoutout to @ToadKing](https://github.com/ToadKing).
+1. Get the device id: `lsusb | grep "SteelSeries Apex"`:
+
+```shell
+$ lsusb | grep "SteelSeries Apex"
+Bus 001 Device 002: ID 1038:1610 SteelSeries ApS SteelSeries Apex Pro
+```
+
+The **id** is the left part of the ID.
 
-If those don't work and lead to an "Access denied" error please try the rules by [FrankGrimm](https://github.com/FrankGrimm) from [here](https://github.com/FrankGrimm/apex7tkl_linux).
+2. Enter the following data from [here](https://gist.github.com/ToadKing/d26f8f046a3b707e9e4b9821be5c9efc) (Shoutout [to @ToadKing](https://github.com/ToadKing)).
```

Comment:
I didn't see that, that solve my other comment, but why is it after the lsusb ? I don't think executing lsusb is a requirement ?

Reply:
where is `lsusb`?

Reply:
the 1. step